components/Layout/AudioControls.vue

```diff
@@ -1,51 +1,122 @@
+                      ? 'mdi-heart'
+                      : 'mdi-heart-outline'
+                  }}</v-icon>
+                </v-btn>
+              </span>
+              <nuxt-link
+                tag="span"
+                class="text--secondary text-caption text-truncate mt-md-n2 link"
+                :to="`/artist/${getCurrentItem().AlbumArtists[0].Id}`"
+              >
+                {{ getCurrentItem().AlbumArtist }}
+              </nuxt-link>
+            </div>
+          </v-col>
+          <v-col cols="6" class="pa-0 d-none d-md-inline">
+            <div class="d-flex flex-column justify-center">
+              <div class="d-flex align-center justify-center">
+                <v-btn
+                  icon
+                  :input-value="isShuffling"
+                  class="mx-1"
+                  @click="toggleShuffle"
+                >
+                  <v-icon>{{
+                    isShuffling ? 'mdi-shuffle' : 'mdi-shuffle-disabled'
+                  }}</v-icon>
+                </v-btn>
+                <v-btn icon class="mx-1" @click="setPreviousTrack">
+                  <v-icon>mdi-skip-previous</v-icon>
+                </v-btn>
+                <v-btn icon large class="mx-1" @click="togglePause">
+                  <v-icon large>
+                    {{ isPaused ? 'mdi-play' : 'mdi-pause' }}
+                  </v-icon>
+                </v-btn>
+                <v-btn icon class="mx-1" @click="stopPlayback">
+                  <v-icon>mdi-stop</v-icon>
+                </v-btn>
+                <v-btn icon class="mx-1" @click="setNextTrack">
+                  <v-icon>mdi-skip-next</v-icon>
+                </v-btn>
+                <v-btn icon class="mx-1" @click="toggleRepeatMode">
+                  <v-icon>{{ repeatIcon }}</v-icon>
+                </v-btn>
+              </div>
+              <time-slider v-if="!isFullScreenPlayer" />
+            </div>
+          </v-col>
+          <v-col cols="3" class="d-none d-md-flex align-center justify-end">
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn disabled icon class="mr-2" v-bind="attrs" v-on="on">
+                  <v-icon>mdi-playlist-play</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('queue') }}</span>
+            </v-tooltip>
+            <volume-slider />
+            <transition name="fade-fast" mode="in-out">
+              <v-tooltip v-if="!isFullScreenPlayer" top>
+                <template #activator="{ on, attrs }">
+                  <nuxt-link tag="span" :to="'/playback'">
+                    <v-btn icon class="ml-2" v-bind="attrs" v-on="on">
+                      <v-icon>mdi-fullscreen</v-icon>
+                    </v-btn>
+                  </nuxt-link>
+                </template>
+                <span>{{ $t('fullScreen') }}</span>
+              </v-tooltip>
+            </transition>
```

Comment:
What is wrong with expanding the modal to full screen?

Reply:
@Camc314 Swiper's weirdness, DOM performance (slides are expensive, and with items in the background, much worse) and a bug in Vuetify where the scrollbar doesn't disappear and allowed you to scroll the things that were below the v-dialog.